src/login/AzureLoginHelper.ts

```diff
@@ -97,6 +90,8 @@ export class AzureLoginHelper {
 	};
 
 	constructor(private context: ExtensionContext, private reporter: TelemetryReporter) {
+		this.authProvider = new AbstractAuthProvider(context, enableVerboseLogs);
```

Comment:
The constructor of an abstract class is generally not accessible. You would do `new AdalAuthProvider()` or `new MsalAuthProvider()` and use those directly